core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/AmbariClusterConnector.java

```diff
@@ -231,6 +231,13 @@ private void rollbackUpscaledInstances(Stack stack, Cluster cluster, AmbariClien
         }
     }
 
+    public AmbariHosts getAmbariHosts(Stack stack) throws CloudbreakSecuritySetupException {
```

Comment:
I know this class was created earlier but please rename it to AmbariHostsCheckerContext for clarity.